websites/L/Leosight/presence.ts

```diff
@@ -0,0 +1,459 @@
+const presence = new Presence({
+    clientId: "888141162488143893"
+  }),
+  time = Math.floor(Date.now() / 1000);
+let item, item2;
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "lslogo",
+      smallImageKey: "lsminilogo",
+      smallImageText: "leosight.cz"
+    },
+    path = document.location.pathname;
+  presenceData.startTimestamp = time;
+
+  //filtrace fóra//
```

Comment:
What does this mean? Is it neccessary?

Reply:
It defines presenceData.startTimestamp to value time, which is defined in the 2nd line of code, i think. But it isn't necessary to have variable time, I can define it into presenceData.StartTimestamp.

Reply:
Obviously I know what that does, but I was talking about L15

Reply:
Ah, it was just for me to know, it isn't neccessary.